frontend/add-client/add-client.component.tsx

```diff
@@ -19,7 +19,9 @@ import { mediaMobile, mediaDesktop } from "../styleguide.component";
 
 export default function AddClient(props: AddClientProps) {
   const scope = useCss(css);
-  const [step, setStep] = useState<Step>(Step.CHECK_DUPLICATE);
+
+  // FIXME - change back check duplicate
+  const [step, setStep] = useState<Step>(Step.CONTACT_INFORMATION);
```

Comment:
This should be reverted back to CHECK_DUPLICATE

Reply:
Should be fixed now